vllm_ascend/worker/model_runner_v1.py

```diff
@@ -1,5 +1,7 @@
 #
 # Copyright (c) 2025 Huawei Technologies Co., Ltd. All Rights Reserved.
+# This file is a part of the vllm-ascend project.
```

Comment:
There are too many useless change in this PR. Please only change the code you need. Do not do some useless change.  For example why move these two lines here?